.travis.yml

```diff
@@ -18,6 +18,7 @@ install:
 - python bootstrap.py
 - bin/buildout buildout:test-eggs=$TEST_EGGS annotate
 - bin/buildout buildout:test-eggs=$TEST_EGGS
+- pip install requests
```

Comment:
I would had the requests package here instead of installing it with pip: 

- https://github.com/collective/collective.behavior.amp/blob/master/setup.py#L42